cmd/removeEnumerator.go

```diff
@@ -75,6 +71,9 @@ func newRemoveEnumerator(cca *CookedCopyCmdArgs) (enumerator *CopyEnumerator, er
 	if cca.IncludeAfter != nil {
 		filters = append(filters, &IncludeAfterDateFilter{Threshold: *cca.IncludeAfter})
 	}
+	if !cca.trailingDot && (cca.FromTo.To() == common.ELocation.File() || cca.FromTo.From() == common.ELocation.File()) {
+		filters = append(filters, &TrailingDotFilter{})
```

Comment:
open question: how to deal with files that have trailing dot if trailing dot is off